pipenv/utils.py

```diff
@@ -697,24 +697,27 @@ def pip_command(self):
             self._pip_command = self._get_pip_command()
         return self._pip_command
 
-    def prepare_pip_args(self, use_pep517=False, build_isolation=True):
+    def prepare_pip_args(self, use_pep517=False, build_isolation=True, pre=False):
         pip_args = []
         if self.sources:
             pip_args = prepare_pip_source_args(self.sources, pip_args)
         if use_pep517 is False:
             pip_args.append("--no-use-pep517")
         if build_isolation is False:
             pip_args.append("--no-build-isolation")
+        if pre is True:
```

Comment:
```suggestion
        if self.pre:
```

So you don’t need to pass the value around.